Microsoft.Toolkit.Uwp.UI.Controls.Core/InAppNotification/InAppNotification.cs

```diff
@@ -254,51 +272,54 @@ private void UpdateContent(NotificationOptions notificationOptions)
         /// Handle the display of the notification based on the current StackMode
         /// </summary>
         /// <param name="notificationOptions">Information about the notification to display</param>
-        private void Show(NotificationOptions notificationOptions)
+        private async void Show(NotificationOptions notificationOptions)
         {
-            var eventArgs = new InAppNotificationOpeningEventArgs();
-            Opening?.Invoke(this, eventArgs);
-
-            if (eventArgs.Cancel)
+            await Dispatcher.RunAsync(Windows.UI.Core.CoreDispatcherPriority.Normal, () =>
```

Comment:
Was this causing an issue somewhere else? Didn't see this called out in the PR notes.

In either case it needs to be `DispatcherQueue` now. 🙂 Dispatcher is no longer a thing we should use. We have the DispatcherQueue helpers we use in other controls too, but if InAppNotification doesn't have a reference already, this may get a bit messy.

Reply:
Crap just noticed we have DispatcherTimer at the top too, it should probably be DispatcherQueueTimer... I'm assuming @azchohfi made these updates in the WinUI 3 branch, I'll check quick.

Reply:
Nope, I'm seeing a lot of Dispatchers still in the code, thought we had fixed them all early on. I'll file an issue to track these later. Think ThemeListener is the best pattern we have currently?

Reply:
Let me try removing them and see how it goes. I tend to add them during debugging to see if the XAML timer is causing any issues. They may not be necessary at all tbh

Reply:
Gotcha! I've made the fixes for InAppNotification. Works well :) Thanks for the hint on ThemeListener, it was a good example.